lighthouse-core/test/config/config-test.js

```diff
@@ -8,7 +8,7 @@
 const Config = require('../../config/config');
 const assert = require('assert');
 const path = require('path');
-const defaultConfig = require('../../config/default-config.js');
+const defaultConfig = require('../../config/default-config');
```

Comment:
:( :(

Reply:
oooooh wait I think it's trying to match the ones around it! so clever :D